### ISSUE TITLE
Price tags did not fit on the schedule

### ISSUE THREAD

**Issue by serzhikdnepr** [OpenFlutter/k_chart#59]
Is it possible to make sure that the price tags do not fit on the chart, or can the chart itself be shifted to the left of the price tags
Like in video
 https://raw.githubusercontent.com/gwhcn/flutter_k_chart/master/example/images/k_chart.2019-09-01%202010_19_56.gif

**Comment by mafanwei** [comment]
I'm sorry I didn't understand.

**Comment by serzhikdnepr** [comment]
Whatever the schedule fits into the numbers
[https://i.postimg.cc/g2b0NTSw-/Imagetrtrt.png](url)

**Comment by mafanwei** [comment]
You mean, let the numbers float above the table?

**Comment by serzhikdnepr** [comment]
Yes, or the ability to shift the graph to the left, as in the first video

**Comment by mafanwei** [comment]
Will do.

**Comment by serzhikdnepr** [comment]
Good afternoon!
Is there any time frame for improvements?

**Comment by mafanwei** [comment]
I have checked and the price tag is on the left side and floats on top so there is nothing to do

**Comment by mafanwei** [comment]
I may not understand what you mean, you can express more clearly?

**Comment by serzhikdnepr** [comment]
![image](https://user-images.githubusercontent.com/6642142/163433626-1920c8c8-8581-44fa-8b4b-758e916e7c5a.png)

**Comment by serzhikdnepr** [comment]
like this
![image](https://user-images.githubusercontent.com/6642142/163433732-aa9709d1-5ac2-4b63-8b50-3b5569e8efae.png)

**Comment by mafanwei** [comment]
Understood
